spec/lib/reporting/protocols_report_spec.rb

```diff
@@ -113,6 +113,38 @@
     it 'generates the tabular csv data' do
       expect(report.as_tables).to eq expected_tables
     end
+
+    describe 'queries' do
+      let(:client) { report.cloudwatch_client }
+      let(:time_query) do
+        {
+          from: report.time_range.begin,
+          to: report.time_range.end,
+        }
+      end
+      before do
+        allow(client).to receive(:fetch).and_call_original
+      end
```

Comment:
Does this do anything?

Reply:
yah, it sets up the spy!

Reply:
Ooh, I misread the setup below as `expect(client).to receive`, thinking that made this `allow` redundant.